packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -1,6 +1,7 @@
 import find from 'lodash/find';
 import filter from 'lodash/filter';
 import get from 'lodash/get';
+import reduce from 'lodash/reduce';
```

Comment:
As the `reduce` function is native javascript code, it is preferable to use it rather than importing the `lodash` function